### PR TITLE
Fix broken pipelines

### PR DESCRIPTION
**Pull request by m7pr**
Fix https://github.com/insightsengineering/teal.osprey/issues/279

### CODE REVIEW

**Comment by m7pr** [comment]
Since we provided a fix in https://github.com/insightsengineering/r-revdepcheck-action/pull/4 for the last build, I am opening this PR and restarting the build

**Comment by m7pr** [comment]
Checks are failiing as there is external cairo dependency missing. Raised this to IDR for help

The same as for teal.goshawk https://github.com/insightsengineering/teal.goshawk/pull/306#issuecomment-2390925941

File: R/tm_g_ae_oview.R
Comment (pawelru) [inline R/tm_g_ae_oview.R:1]: can we remove the `library(nestcolor)` instead?
The functionality should pick up the palette from `nestcolor` if its available and if not - fallback to some default value. That was the initial idea at least.

Plus: we will be having a simpler example code.

File: DESCRIPTION [hunk 1]
```diff
@@ -34,6 +34,7 @@ Imports:
     lifecycle (>= 0.2.0),
     shinyvalidate,
     teal.code (>= 0.4.1.9009),
+    teal.data (>= 0.3.0.9018),
```
Comment (m7pr) [inline DESCRIPTION:37]: It is used in the codebase

https://github.com/search?q=repo%3Ainsightsengineering%2Fteal.osprey%20teal.data%3A%3A&type=code

**Comment by m7pr** [comment]
Running r-hub test https://github.com/insightsengineering/teal.osprey/actions/runs/11232452601

File: NEWS.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # teal.osprey 0.1.16.9028
 
 * Removed `Show Warnings` modals from modules.
+* Moved `teal_data` from `Suggests` to `Imports` as it was used in `tm_g_swimlane`. Changed examples from `cdisc_data()` to `teal_data()`.
```
Comment (pawelru) [inline NEWS.md:4]:
```suggestion
* Moved `teal_data` from `Suggests` to `Imports` as it was used in `tm_g_swimlane`. 
* Changed examples from `cdisc_data()` to `teal_data()`.
```

File: NEWS.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
 # teal.osprey 0.1.16.9028
 
 * Removed `Show Warnings` modals from modules.
-* Moved `teal_data` from `Suggests` to `Imports` as it was used in `tm_g_swimlane`. Changed examples from `cdisc_data()` to `teal_data()`.
+* Moved `teal_data` from `Suggests` to `Imports` as it was used in `tm_g_swimlane`.
```
Comment (pawelru) [inline NEWS.md:4]:
```suggestion
* Moved `teal.data` from `Suggests` to `Imports` as it was used in `tm_g_swimlane`. 
```

File: NEWS.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 # teal.osprey 0.1.16.9028
 
 * Removed `Show Warnings` modals from modules.
+* Moved `teal.data` from `Suggests` to `Imports` as it was used in `tm_g_swimlane`.
```
Comment (pawelru) [inline NEWS.md:4]:
```suggestion
* Moved `teal.data` from `Suggests` to `Imports`. 
```

**Pull request merged**